codebrag-service/src/main/scala/com/softwaremill/codebrag/eventstream/InternalSerializers.scala

```diff
@@ -0,0 +1,61 @@
+package com.softwaremill.codebrag.eventstream
+
+import com.softwaremill.codebrag.domain.reactions.{UnlikeEvent, LikeEvent}
+import com.softwaremill.codebrag.domain._
+import org.json4s.ext.JodaTimeSerializers
+import org.json4s.{DefaultFormats, FieldSerializer}
+import org.json4s.FieldSerializer._
+
+object InternalSerializers {
```

Comment:
Yes, but those serializers define how to serialize events or entities into JSON not hooks., so they can be used with any other future JSON serialization.